packages/sdks/e2e/sveltekit/src/routes/[...catchall]/+page.svelte

```diff
@@ -1,12 +1,57 @@
 <script lang="ts">
   import { Content } from '@builder.io/sdk-svelte';
+  import BuilderBlockWithClassName from '../../components/BuilderBlockWithClassName.svelte';
+  // this data comes from the function in `+page.server.ts`, which runs on the server only
   export let data;
+  const builderBlockWithClassNameCustomComponent = {
+    name: 'BuilderBlockWithClassName',
+    component: BuilderBlockWithClassName,
+    shouldReceiveBuilderProps: {
+      builderBlock: true,
+      builderContext: true,
+      builderComponents: true,
+    },
+    inputs: [
+      {
+        name: 'content',
+        type: 'uiBlocks',
+        defaultValue: [
+          {
+            '@type': '@builder.io/sdk:Element',
+            '@version': 2,
+            id: 'builder-c6e179528dee4e62b337cf3f85d6496f',
+            component: {
+              name: 'Text',
+              options: {
+                text: 'Enter some text...',
+              },
+            },
+            responsiveStyles: {
+              large: {
+                display: 'flex',
+                flexDirection: 'column',
+                position: 'relative',
+                flexShrink: '0',
+                boxSizing: 'border-box',
+                marginTop: '20px',
+                lineHeight: 'normal',
+                height: 'auto',
+              },
+            },
+          },
+        ],
+      },
+    ],
+  };
 </script>
 
 <main>
   {#if data.props}
     {#key data.props}
-      <Content {...data.props} customComponents={data.customComponents} />
+      <Content
+        {...data.props}
+        customComponents={[builderBlockWithClassNameCustomComponent]}
```

Comment:
in here you are overriding `customComponents` coming from `data.props`

you need to add the component here https://github.com/BuilderIO/builder/blob/8b372561e2f8e1cd3c5648a71183934d35992f11/packages/sdks/e2e/sveltekit/src/routes/%5B...catchall%5D/+page.ts#L33-L44